---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    entando.org/entando-resource-namespace: "my-namespace"
    TestResource: "my-app"
    entando.org/entando-resource-kind: "TestResource"
  name: "my-app-13067699"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "test.org/v1"
    kind: "TestResource"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "cc665b84-59a4-484e-b677-6c173d158152"
action: "FAILED"
count: 1
firstTimestamp: "2021-11-26T13:39:35Z"
involvedObject:
  apiVersion: "test.org/v1"
  kind: "TestResource"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "4024011"
  uid: "cc665b84-59a4-484e-b677-6c173d158152"
lastTimestamp: "2021-11-26T13:39:35Z"
message: "The deployment of TestResource my-namespace/my-app failed due to Deployment\
  \ failed. Please inspect the logs of the pod my-namespace/my-app-deployment. Fix\
  \ the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "test-controller-pod"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-ource-controller"
type: "Error"
